mapgen.lua

```diff
@@ -886,7 +886,7 @@ minetest.register_abm({
 	function(pos, node)
 		for _,obj in ipairs(minetest.get_objects_inside_radius(pos, 5)) do
 			if obj:is_player() then
-				explosion(pos, 3, 0, 1)
+				tnt.boom(pos,{radius = 3,dmage_radius = 0})
```

Comment:
`damage_radius` instead of `dmage_radius` - nearly there ;-)

Actually, even if the variables are mis-spelled, this would not "fail":

When the `tnt.boom()` function executes, it looks for a `radius` property in the argument object. If it finds one, that property is used; if not (because the property does not exist, or the property was actually mis-spelled), then a default value is used.

So if you call `tnt.boom(pos, {rodeo = 3, double_rodeo = 0})` (`<-` deliberately nonesense variables), then you will not get an error. You will simply see that your parameters have been ignored, and TNT will boom and damage in a normal way.

Reply:
it makes sense, now i understand the commit i referenced, thanks for that @taikedz ..